service/src/main/java/aero/minova/cas/service/SecurityService.java

```diff
@@ -27,10 +25,9 @@
 import aero.minova.cas.api.domain.Value;
 import aero.minova.cas.controller.SqlViewController;
 import aero.minova.cas.sql.SystemDatabase;
+import lombok.Setter;
```

Comment:
Wird Setter verwendet?

Reply:
Ja, im SqlViewController:

`
	@PostConstruct
	public void init() {
		securityService.setSvc(this);
	}
`